.github/workflows/ci-base.yml

```
@@ -0,0 +1,46 @@
+name: ci-base
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - packages/base/**
+      - ./.github/workflows/ci-base.yml
+  pull_request:
+    branches: [ master ]
+    paths:
+      - packages/base/**
+      - ./.github/workflows/ci-base.yml
+jobs:
```

[same for all workflows]
```suggestion
      - ./.github/workflows/ci-base.yml

jobs:
```